pkg/ddl/ddl_history.go

```diff
@@ -150,9 +152,15 @@ func GetAllHistoryDDLJobs(m *meta.Meta) ([]*model.Job, error) {
 func ScanHistoryDDLJobs(m *meta.Meta, startJobID int64, limit int) ([]*model.Job, error) {
 	var iter meta.LastJobIterator
 	var err error
+
 	if startJobID == 0 {
+		// if 'start_job_id' == 0 and 'limit' == 0(default value), get the last 1024 ddl history job by defaultly.
+		if limit == 0 {
```

Comment:
this changes HTTP API, please confirm with PM @frank945946

please also update doc in here
https://github.com/pingcap/tidb/blob/master/docs/tidb_http_api.md

Reply:
> this changes HTTP API, please confirm with PM @Frank945946
> 
> please also update doc in here https://github.com/pingcap/tidb/blob/master/docs/tidb_http_api.md

Get it~

Reply:
I have already confirmed with @Frank945946， and confirmed with either BR team or CD team.